### ISSUE TITLE
Dropdown: Keyboard controls don't work

### ISSUE THREAD

**Issue by rfire12** [primefaces/primereact#4482]
### Describe the bug

When using the the Dropdown component, I would expect it to work using the keyboard (tab, up arrow, down arrow), but this doesn't seem to work.

When the dropdown panel is open, pressing the "tab" key takes the focus away and pressing the "down arrow" key does nothing.

Currently, the only way to select the dropdown options is using the mouse.

### Reproducer

https://codesandbox.io/s/vigorous-cookies-cjoc0i?file=/src/App.js

### PrimeReact version

9.5.0

### React version

18.x

### Language

ALL

### Build / Runtime

Next.js

### Steps to reproduce the behavior

1. Go to https://codesandbox.io/s/vigorous-cookies-cjoc0i
2. Click on the Dropdown component, or alternatively focus the Dropdown and press the "space" key.
3. When the panel is open, press the "down arrow" key. 
4. As you'd see, nothing happens by pressing the "down arrow" key

### Expected behavior

I should be able to select the options of the Dropdown component using the arrow keys or the tab key.

**Comment by melloware** [comment]
@rfire12 Dropdown will be getting all accessibility traits from PrimeVue in a future version: https://primevue.org/dropdown/#accessibility

Tab key only naivgates to the dropdown icon but once the panel is open Tab would take you to the next component.  Only the arrow keys navigate and ENTER/SPACE selects the item

**Comment by ubau** [comment]
@rfire12 , code in your example is not complete.
Here is working example:
```js
export default function App() {
  const [value, setValue] = useState("") ;
  return (
    <div className="App">
      <Dropdown
        optionValue="value"
        optionLabel="label"
        options={[
          { label: "foo 1", value: "foo1" },
          { label: "foo 2", value: "foo2" },
          { label: "foo 3", value: "foo3" },
          { label: "foo 4", value: "foo4" }
        ]}
        value={value}
        onChange={(e) => setValue(e.value)}
      />
    </div>
  );
}
```

**Comment by habubey** [comment]
Issue tracker is used for defects only as part of our commitment to quality and continuous improvement in all areas. Enhancements are collected as valuable community feedback and managed internally so moving this enhancement ticket to our internal project management backlog.
